api/src/index.ts

```diff
@@ -154,6 +155,22 @@ async function main() {
 
   getDiscordClient();
 
+  const pgClient = await connectToDatabase(); //? maybe we should not keep this dependency in the discord folder?
```

Comment:
I want to do a cleanup of the Discord integration when I start working on https://github.com/TFNS/CTFNote/issues/239 and then I will reorganize this too.